walkthroughs/week-2-server/examples/random-quotes/src/main/webapp/index.html

```diff
@@ -10,7 +10,7 @@
     <div id="content">
```

Comment:
You can probably revert all the files within week-2-sever/examples and week-3-server/examples, as they are not part of your webpage.